README.md

```diff
@@ -26,10 +26,22 @@ If you are migrating a service to DBT PaaS, [GOV.UK PaaS to DBT PaaS Migration](
 
 ### Testing
 
+#### Automated testing
+
 Run `poetry run pytest` in the root directory to run all tests.
 
 Or, run `poetry run tox` in the root directory to run all tests for multiple Python versions. See the [`tox` configuration file](tox.ini).
 
+#### Manual testing
+
+You may want to test any CLI changes locally. 
+
+Run `poetry build` to build your package resulting in a package file (e.g. `dbt_copilot_tools-0.1.40.tar.gz`) in a `dist` folder. You may need to bump up the package version before doing so.
+
+Copy the package file(s) to a repository where you would like to test your changes, and make sure you are in a virtual environment.
+
+Run `pip install <file>` and confirm the installation has worked by running `copilot-helper --version` which would output version `0.1.40` following our example.
```

Comment:
Maybe add a note here about ensuring that you revert to the published version after testing in a repo to make sure you're on the right version when you are working with the repo normally.

Reply:
Good point. Let me add another paragraph at the end after manual testing is complete.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/171/commits/02ac3957c216ed8f024cbf9c8a12dbfc5b33394d.